pkg/controller/machinepool/vsphereactuator.go

```diff
@@ -105,3 +123,27 @@ func getVSphereOSImage(masterMachine *machineapi.Machine, scheme *runtime.Scheme
 	logger.WithField("image", osImage).Debug("resolved image to use for new machinesets")
 	return osImage, nil
 }
+
+func setComputeClusterPath(cluster, datacenter string, logger log.FieldLogger) string {
+	if cluster != "" && !strings.HasPrefix(cluster, "/") {
+		logger.Warn("computeCluster as a non-path is now depreciated please use the form: /%s/host/%s", datacenter, cluster)
+		return fmt.Sprintf("/%s/host/%s", datacenter, cluster)
+	}
+	return cluster
+}
+
+func setDatastorePath(datastore, datacenter string, logger log.FieldLogger) string {
+	if datastore != "" && !strings.HasPrefix(datastore, "/") {
+		logger.Warn("datastore as a non-path is now depreciated please use the form: /%s/datastore/%s", datacenter, datastore)
+		return fmt.Sprintf("/%s/datastore/%s", datacenter, datastore)
+	}
+	return datastore
+}
+
+func setFolderPath(folder, datacenter string, logger log.FieldLogger) string {
+	if folder != "" && !strings.HasPrefix(folder, "/") {
+		logger.Warn("folder as a non-path is now depreciated please use the form: /%s/vm/%s", datacenter, folder)
+		return fmt.Sprintf("/%s/vm/%s", datacenter, folder)
+	}
+	return folder
+}
```

Comment:
Ugh. But yeah. Please add a comment: "copied from https://github.com/openshift/installer/blob/f7731922a0f17a8339a3e837f72898ac77643611/pkg/types/vsphere/conversion/installconfig.go#L75-L97" kind of thing.